.ci_support/compute_build_graph.py

```diff
@@ -39,15 +39,13 @@
 import functools
```

Comment:
Please revert changes to this file. You shouldn't need to modify anything outside of `recipes/conda-which`